src/ui/app/html_template.ejs

```diff
@@ -12,7 +12,11 @@
     <title>Cassandra Reaper<%= htmlWebpackPlugin.options.title %></title>
 
     <!-- <base href="/webui/"> -->
-    <base href="<%= htmlWebpackPlugin.options.baseUrl %>" />
+    <script type="text/javascript">
+        var pathName = document.location.pathname;
+        var contextPath = pathName.substring(0, pathName.indexOf("<%= htmlWebpackPlugin.options.baseUrl %>"));
+        document.write('<base href="' + contextPath + '<%= htmlWebpackPlugin.options.baseUrl %>" />');
+    </script>
```

Comment:
The contextPath will not be correctly constructed if there's a `/` at the end of the baseUrl, so it breaks a change I've made lately to avoid having a blank page when going to `/webui` instead of `/webui/`.
Here's what I've changed to make it work in all situations : 

```
<script type="text/javascript">
        var pathName = document.location.pathname;
        var baseUrl = "<%= htmlWebpackPlugin.options.baseUrl %>".endsWith('/') ? "<%= htmlWebpackPlugin.options.baseUrl %>".substring(0,"<%= htmlWebpackPlugin.options.baseUrl %>".lastIndexOf('/')):"<%= htmlWebpackPlugin.options.baseUrl %>";
        var contextPath = pathName.substring(0, pathName.indexOf(baseUrl));
        document.write('<base href="' + contextPath + '<%= htmlWebpackPlugin.options.baseUrl %>" />');
    </script>
```

What do you think of it ?

Reply:
Ahh, sorry about missing that.  Your alteration looks perfect.  Thank you